### PR TITLE
Select postcode

### PR DESCRIPTION
**Pull request by vincit-matu**
Updates postal code selection in housing company creation

- adds relevant models and queries
- also fixes some react warnings (left over/out code)

### CODE REVIEW

File: frontend/src/common/models.ts [hunk 1]
```diff
@@ -181,6 +181,12 @@ export interface PageInfo {
     };
 }
 
+export interface IPostalCode {
+    value: string;
+    city: string | null;
+    cost_area: string;
```
Comment (toikarin-vincit) [inline frontend/src/common/models.ts:187]: this is an integer between 1-4

File: frontend/src/common/models.ts [hunk 2]
```diff
@@ -181,6 +181,12 @@ export interface PageInfo {
     };
 }
 
+export interface IPostalCode {
+    value: string;
+    city: string | null;
```
Comment (toikarin-vincit) [inline frontend/src/common/models.ts:186]: this cannot be `null`

Comment (vincit-matu) [reply frontend/src/common/models.ts]: Updated accordingly

File: frontend/src/common/models.ts (hunk 1 above)
Comment (vincit-matu) [reply frontend/src/common/models.ts]: Updated to be of type 1 | 2 | 3 | 4.

**Review by toikarin-vincit — APPROVED** [review]

**Pull request merged**